http-transport-binding.md

```diff
@@ -330,6 +340,105 @@ Content-Length: nnnn
 
 ```
 
+### 3.3. Batched Content Mode
+
+In the *batched* content mode several events are batched into a single HTTP
+request or response body. The chosen [event format](#14-event-formats) MUST
+define how a batch is represented. Currently, the only format supporting
```

Comment:
You have to change "currently, the only" if/when that changes. It makes sense to make an absolute statement like "The JSON event format that MUST be supported by any compliant implementation supports batching."

Reply:
I didn't implement your suggestion exactly, because it seems to imply that batching MUST be supported by any compliant implementation. In the current proposal, it is however optional, and I think it should stay optional, as I wrote in the PR description:

> I also did not include the batching mode as one of the modes the receiver SHOULD support. Take a Function as a Service: Usually, each event should be processed in on an own instance of a function. This is trivial to implement when a HTTP Request always contains a single event. A batch doesn't map nicely onto the core FaaS abstractions.

Is what I added in https://github.com/cloudevents/spec/pull/370/commits/1191dd226eeb20d77a4d99f47ce6487065e28995 fine?